test/tutorial/test_README_example.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iomanip>
```

Comment:
If we wanted, we could have cmake write the content for this test and the readme. If we ever changed the contents of that cmake file, the tests and the readme would be updated together. That's probably more work than is necessary though

Reply:
maybe revisit this when we start writing tutorials in the documentation?

Reply:
fine with me